x-pack/packages/kbn-cloud-security-posture-common/schema/rules/v3.ts

```diff
@@ -37,6 +37,7 @@ export const cspBenchmarkRuleMetadataSchema = schema.object({
   profile_applicability: schema.string(),
   rationale: schema.string(),
   references: schema.maybe(schema.string()),
+  reference: schema.maybe(schema.string()),
```

Comment:
a suggestion: maybe it would be simpler to fix it in the ingest pipeline and relay on it here?

Reply:
@kfirpeled do you mean in the ingest pipeline of `cloud_security_posture` copy the value of `rule.references` into `rule.reference` to be able to remove `rule.references` completely? We could do that, but as the change to support both is quite small, I thought it would be safer just to support both at first and then depreciate `rule.references`. Otherwise I need to think how the UI will work out with the older version of the integration, as integration update is not a given when you update the stack version. Am I missing smth?

Reply:
nope, you are not missing anything. 
However, since we do have automatic updates of the integration, after 24 hours all the misconfigurations will have the updated field. so it shortens your migration process.

both ways work